src/main/java/nextstep/subway/line/dto/LineResponse.java

```diff
@@ -0,0 +1,55 @@
+package nextstep.subway.line.dto;
+
+import nextstep.subway.line.domain.Line;
+import nextstep.subway.station.dto.StationResponse;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LineResponse {
+    private Long id;
+    private String name;
+    private String color;
+    private List<StationResponse> stations;
+
+    protected LineResponse() {
+    }
+
+    public LineResponse(Long id, String name, String color) {
+        this.id = id;
+        this.name = name;
+        this.color = color;
+    }
+
+    public LineResponse(Long id, String name, String color, List<StationResponse> stations) {
+        this.id = id;
+        this.name = name;
+        this.color = color;
+        this.stations = stations;
+    }
+
+    public static LineResponse of(Line line) {
+        List<StationResponse> stationResponses = line.getSections().getStationsSorted()
```

Comment:
```suggestion
        List<StationResponse> stationResponses = line.getStationsSorted()
```

getter를 연속해서 사용하는 부분을 제거해보면 좋겠어요~